front/app/components/Form/Components/Layouts/utils.ts

```diff
@@ -40,6 +40,37 @@ export interface PageCategorization extends ExtendedUISchema {
   elements: (PageType | PageCategorization)[];
 }
 
+export const keyPresentInPageRoute = (
+  key: string,
+  userPageRoute: PageType[]
+) => {
+  let isFound = false;
+  userPageRoute.map((page) => {
+    const currentPageElementNames = page.elements.map((uiSchemaElement) =>
+      uiSchemaElement.scope.split('/').pop()
+    );
+    if (currentPageElementNames) {
+      if (currentPageElementNames.find((elementName) => elementName === key)) {
+        isFound = true;
+      }
```

Comment:
Can't you use `includes` here?
```suggestion
      isFound = currentPageElementNames.includes(key)
```

Reply:
For this case I don't think so, since it's within the context of a page map, so even if it assigns isFound to true at some point, it could be overwritten by false if it's not present in the next loop :thinking:

Reply:
I am stupid. Indeed. It should have been:
```
isFound ||= currentPageElementNames.includes(key)
```

Reply:
Ah I've never used `||= ` before :) Makes sense - I'll use that. Thanks!!